File: ui/src/androidTest/java/com/xmartlabs/bigbang/ui/recyclerview/singleitem/SimpleItemTestSetItems.java
package com.xmartlabs.bigbang.ui.recyclerview.singleitem;

import com.xmartlabs.bigbang.ui.recyclerview.common.Car;

import org.junit.Test;

import java.util.List;

public class SimpleItemTestSetItems extends SimpleItemRecyclerViewTest {
  @Test
  public void testSetItems() {
    List<Car> cars = getCarList();

    SingleItemActivity activity = activityRule.getActivity();
    activity.runOnUiThread(() -> activity.getAdapter().setItems(cars));

    checkItems(cars);
  }
}

File: ui/src/androidTest/java/com/xmartlabs/bigbang/ui/recyclerview/singleitem/SimpleItemTestSetItems.kt
package com.xmartlabs.bigbang.ui.recyclerview.singleitem

import org.junit.Test

class SimpleItemTestSetItems : SimpleItemRecyclerViewTest() {
  @Test
  fun testSetItems() {
    val cars = carList

    val activity = activityRule.activity
    activity.runOnUiThread { activity.adapter.setItems(cars) }

    checkItems(cars)
  }
}
